code/style.css

```diff
@@ -1,4 +1,710 @@
-html {
-  background: #0025ff;
-  color: #fff;
+/* fonts imported from google fonts */
+@import url('https://fonts.googleapis.com/css2?family=Kufam:wght@500&family=Roboto&family=Orbitron:wght@500&display=swap');
+
+/* body is set to relative so footer can be properly aligned by using this parent. 
+A min-height and padding same height as footer is set to be sure that footer stays on bottom of page even when there is no main content added */
+body {
+  margin: 0;
+  padding: 0;
+  background-color: rgba(2, 24, 44, 0.897);
+  color: rgb(11, 102, 155);
+  position: relative;
+  padding-bottom: 40px;
+  min-height: 100vh;
 }
+
+/* Meteor styling */
+.meteor {
+  border-color: rgba(231, 116, 71, 0);
+  background-color: rgba(235, 100, 59, 0.74);
+  border-radius: 50%;
+  box-shadow: 0px 0px 3px 3px rgba(235, 65, 59, 0.74);
+  width: 5px;
+  height: 5px;
+  position: absolute;
+  top: 800px;
+  left: 40%;
+  animation: meteor_path 15s linear infinite;
+}
+
+/* Animation for the meteor going over the sky in the header */
+@keyframes meteor_path {
+  0% {
+    transform: rotate(-140deg) translateX(800px);
+  }
+
+  50% {
+    transform: rotate(-40deg) translateX(800px) scale(3);
+    ;
+  }
+
+  80% {
+    transform: rotate(-40deg) translateX(800px) scale(9);
+  }
+
+  90% {
+    transform: rotate(-40deg) translateX(800px) scale(0.1);
+  }
+
+  100% {
+    transform: rotate(-40deg) translateX(1000px) scale(0.1);
+  }
+}
+
+/* wraps all contents in the header and sets header background picture */
+.header {
+  display: flex;
+  position: relative;
+  justify-content: flex-end;
+  background-image: url("./pictures/ISS.jpg");
+  background-attachment: fixed;
+  background-size: cover;
+  height: 400px;
+  width: 100%;
+  box-shadow: 0 1px 4px 2px #f1df77;
+  background-position-y: -15vh;
+  margin-bottom: 40px;
+  overflow: hidden;
+}
+
+/* a simpler header for sites that are not index-site */
+.simple-header {
+  position: relative;
+  display: flex;
+  justify-content: flex-end;
+  width: 100%;
+  height: 180px;
+  background: rgba(0, 0, 0, 0.575);
+}
+
+/* container for main logo*/
+.logo {
+  width: 280px;
+  height: 144px;
+  position: relative;
+  align-self: flex-end;
+}
+
+.logo img,
+.logo-simple img {
+  width: 100%;
+  max-height: 100%;
+  object-fit: cover;
+}
+
+/*hides the simpler logo used only for mobile device*/
+.logo-simple {
+  display: none;
+  position: relative;
+  align-self: flex-end;
+}
+
+
+/* navigation bar sticky on top of the page, flex is used to place menu content and search bar on the navbar */
+.nav {
+  display: flex;
+  width: 100%;
+  height: 25px;
+  align-items: center;
+  justify-content: space-evenly;
+  border-bottom: solid #f1df77 1px;
+  background-color: rgba(0, 0, 0, 0.425);
+  position: fixed;
+  z-index: 1;
+}
+
+/* necessary for doing drop-down menu that appears on click in mobile version */
+#checkbox,
```

Comment:
We shouldn't style by id's, needs to be fixed as well! (details in feedback document)